src/Log/Engine/BaseLog.php

```diff
@@ -66,15 +67,15 @@ public function __construct(array $config = [])
      *
      * @return array
      */
-    public function levels()
+    public function levels(): array
     {
         return $this->_config['levels'];
     }
 
     /**
      * Get the scopes this logger is interested in.
      *
-     * @return array
+     * @return array|bool
```

Comment:
Could we use null as alternative value?

Reply:
We could but that will break compatibility. Right now `false` is used to indicate 'no scope'. I'd like to revisit scopes/tags in the future so this might go away as part of 4.x releaess.